CMake/Dependencies/libkvscproducer-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvscproducer-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-	GIT_TAG           69f68a7263efc6435c8dcf64ed163c9baa699325
+	GIT_TAG           4fc9d18d709e552f8d0b4f4f83a38c5dab726f86
```

Comment:
let us add the detail on how this works in conjunction with the log4j changes in the Java SDK

Reply:
This git-tag is from the version of producer c with 3.6.3